design/account.go

```diff
@@ -221,6 +222,7 @@ var updateUserDataAttributes = a.Type("UpdateIdentityDataAttributes", func() {
 	a.Attribute("email", d.String, "The email")
 	a.Attribute("bio", d.String, "The bio")
 	a.Attribute("url", d.String, "The url")
+	a.Attribute("email_private", d.Boolean, "Whether the email address would be visible to other users.")
```

Comment:
Same as earlier,
"Whether the email address wouldn't be visible to other users" or "Whether the email address would be private"